VirtualH89/Src/HostFileBdos.h

```diff
@@ -9,61 +9,109 @@
 #ifndef HOSTFILEBDOS_H_
 #define HOSTFILEBDOS_H_
 
-#include "config.h"
-#include "h89Types.h"
 #include "NetworkServer.h"
 #include "propertyutil.h"
 
 #include <dirent.h>
 #include <string>
 
-#define BDOS_FUNC(name)                                            \
-    int name(BYTE * msgbuf, int len);                              \
-    static int name(HostFileBdos * thus, BYTE * msgbuf, int len) { \
-        return thus->name(msgbuf, len);                            \
+#define BDOS_FUNC(name)                                               \
+    int name(uint8_t * msgbuf, int len);                              \
+    static int name(HostFileBdos * thus, uint8_t * msgbuf, int len) { \
+        return thus->name(msgbuf, len);                               \
     }
 
+// An instance of HostFileBdos represents a client-server pair.
+// This is typically a dedicated network connection (e.g. socket),
+// but may also be an informal pairing such as a virtual machine's
+// "connection" to the native host.
+
 class HostFileBdos: public NetworkServer
 {
   public:
     HostFileBdos(PropertyUtil::PropertyMapT& props,
-                 std::vector<std::string> args, BYTE srvId);
+                 std::vector<std::string> args, uint8_t srvId, uint8_t cltId);
     virtual ~HostFileBdos();
 
-    virtual int checkRecvMsg(BYTE clientId, BYTE* msgbuf, int len);
-    virtual int sendMsg(BYTE* msgbuf, int len);
-
+    virtual int checkRecvMsg(uint8_t clientId, uint8_t* msgbuf, int len);
+    virtual int sendMsg(uint8_t* msgbuf, int len);
   private:
-    const char* dir;
-    BYTE        serverId;
+    char*   dir;
+    uint8_t serverId;
+    uint8_t clientId;
 
     struct fcb
     {
-        BYTE drv;
-        BYTE name[11];
-        BYTE ext;
-        BYTE s1[2];
-        BYTE rc;
+        uint8_t drv;
+        uint8_t name[11];
+        uint8_t ext;
+        uint8_t s1[2];
+        uint8_t rc;
         union
         {
-            BYTE d0[16];
+            uint8_t d0[16];
             struct
             {
-                int fd;
-                int oflags;
+                uint16_t fd;
+                uint16_t fd_;
+                int      oflags;
             };
         };
-        BYTE cr;
-        BYTE rr[3];
+        uint8_t cr;
+        uint8_t rr[3];
+    } __attribute__((packed));
+
+    struct fcbdate
```

Comment:
Can you use camelCase for the struct names, like what is used with the variables?